beacon-chain/core/helpers/beacon_committee.go

```diff
@@ -58,6 +59,29 @@ func SlotCommitteeCount(activeValidatorCount uint64) uint64 {
 	return committeesPerSlot
 }
 
+// AttestationCommittees returns beacon state committees that reflect attestation's committee indices.
+func AttestationCommittees(ctx context.Context, st state.ReadOnlyBeaconState, att ethpb.Att) ([][]primitives.ValidatorIndex, error) {
+	var committees [][]primitives.ValidatorIndex
+	if att.Version() >= version.Electra {
+		committeeIndices := att.CommitteeBitsVal().BitIndices()
+		committees = make([][]primitives.ValidatorIndex, len(committeeIndices))
+		var err error
+		for i, ci := range committeeIndices {
+			committees[i], err = BeaconCommitteeFromState(ctx, st, att.GetData().Slot, primitives.CommitteeIndex(ci))
+			if err != nil {
+				return nil, err
+			}
+		}
```

Comment:
Not sure if this version is cleaner. Consistent error handling by declaring committee and err within the loop.

```suggestion
	var committees [][]primitives.ValidatorIndex
	if att.Version() >= version.Electra {
		committeeIndices := att.CommitteeBitsVal().BitIndices()
		committees = make([][]primitives.ValidatorIndex, len(committeeIndices))
		for i, ci := range committeeIndices {
			committee, err := BeaconCommitteeFromState(ctx, st, att.GetData().Slot, primitives.CommitteeIndex(ci))
			if err != nil {
				return nil, err
			}
			committees[i] = committee
		}
```

Reply:
done